manifests/master/simp_auth.pp

```diff
@@ -1,55 +1,48 @@
 # Add SIMP-specific entries to PuppetServer's auth.conf
 #
+# For documentation about *_allow and *_deny, see the puppetserver docs
```

Comment:
I think the reference is useful, but should not replace the descriptions for the 6 added parameters.

Reply:
Good catch here too